File: 1-two-sum/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        values={}
        for i in range(len(nums)):
            number=target-nums[i]
            if number in values:
                return [values[number],i]
            values[nums[i]]=i
            

File: 1-two-sum/1-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int>::iterator it;
        vector<int>answer;
        for(int i=0;i<nums.size();i++)
        {
            it=find(nums.begin(),nums.end(),target-nums[i]);
            if(it!=nums.end()&&it-nums.begin()!=i)
            {
                answer.push_back(i);
                answer.push_back(it-nums.begin());
                return answer;
            }
                
        }
        return answer;
    }
};